step.yml

```diff
@@ -1,13 +1,29 @@
 title: "fastlane"
 summary: |-
-  Runs your fastlane lane.
+  Runs your fastlane lane [bitrise.io](https://www.bitrise.io).
 description: |-
-  Runs your fastlane lane.
+  fastlane is an open-source app automation tool for iOS, Android and for most cross-platform frameworks, for example, React Native, Xamarin and Flutter.
+  **fastlane** Step helps you integrate your lane to your Bitrise Workflow and runs your lane based on the fastlane actions with minimal configuration.
+  
+  ### Configuring the Step
+  1. Add the **fastlane** Step to your Workflow after the **Git Clone Repository** Step or any other dependency Step.
+  1. Based on your project's fastlane setup, you can add your project's default lane or a custom lane in the **fastlane lane** input.
+  2. If your fastlane directory is not available in your repository's root, then you can add the parent directory of fastlane directory in the **Working directory** input.
+  3. If your project doesn't contain a fastlane gem in your project's Gemfile, you can use the **Should update fastlane gem before run** input. Set this input to `true` so that the Step can install the latest fastlane version to your project.
```

Comment:
Would add something like:
If a gem lockfile (Gemfile.lock or gems.locked) in the working directory includes the fastlane gem, that specific fastlane version will be installed.